tests/flow/intersection/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -79,7 +79,8 @@ function hasObjectMode_ok(options: DuplexStreamOptions): boolean {
  * @flow
  */
 
-type DuplexStreamOptions = ReadableStreamOptions & WritableStreamOptions & {
+type DuplexStreamOptions = ReadableStreamOptions &
+  WritableStreamOptions & {
   allowHalfOpen?: boolean,
   readableObjectMode?: boolean,
```

Comment:
This seems slightly unfortunate. The contents of the object line up with previous expressions. Are we sure we want that?